template/default/fulldoc/yaml/setup.rb

```diff
@@ -1,16 +1,10 @@
 def init
   options.serializer = Serializers::FileSystemSerializer.new :extension => "yml"
   options.objects.each do |object|
-    begin
-      next if object.root?
-      serialize(object)
-    rescue => e
-      path = options.serializer.serialized_path(object)
-      log.error "Exception occurred while generating '#{path}'"
-      log.backtrace(e)
-    end
+    next if object.root?
+    serialize(object)
```

Comment:
These appear not to be covered by rubocop yet. We'll have to look at that. But for now, please remove the parens.